components/tabs/src/storage.rs

```diff
@@ -270,8 +270,10 @@ impl TabsStorage {
             Ok(Some(conn)) => conn,
         };
         let pending_tabs_result: Result<Vec<(String, String)>> = conn.query_rows_and_then_cached(
```

Comment:
Nit: Let's unify this and the inactive query you added into the same `SELECT`; it would avoid scanning the table twice, since `command = :command` can't use an index 😅